tests/FunctionalTests/ServerLess/FilteringTest.cs

```diff
@@ -52,8 +52,13 @@ public async Task ItSupportsASingleFilter(string memoryType)
             await Task.Delay(TimeSpan.FromSeconds(2));
         }
 
+        // Simple filter: empty filter returns the memory
+        var answer = await this._memory.AskAsync("What is Orion?", filter: new(), index: indexName);
```

Comment:
@dluc See here for a functional test. I assume the expected response if an empty filter is passed is to assume no filter should apply to the query?

Before the fix you'd get the following error when using ACS:
```
Azure.RequestFailedException : Invalid expression: Expression expected at position 1 in '()'.
Parameter name: $filter
Status: 400 (Bad Request)
```